code/__DEFINES/preferences.dm

```diff
@@ -43,4 +43,4 @@
 #define BE_GANG			4096
 #define BE_SHADOWLING	8192
 #define BE_ABDUCTOR		16384
-#define BE_REVENANT		32768
+#define BE_REVENANT		32768
```

Comment:
Didnt even touch this file
I dunno why git artificially engorges my commits with changes